less/components/result-status.less

```diff
@@ -220,7 +220,6 @@
     background-color: @pre-bg;
     color: @pre-color;
     padding: 0.5em;
-    border-left: solid 28px darken(@pre-bg, 5%);
```

Comment:
I am not sure, @i386 asked that this should be removed. I had an override before for the logConsole. If you you need this I can switch back to the override